src/backend/joanie/core/api/client/__init__.py

```diff
@@ -1468,6 +1468,12 @@ class NestedOrderCourseViewSet(NestedGenericViewSet, mixins.ListModelMixin):
         .distinct()
     )
 
+    def get_queryset(self):
+        """Returns the queryset of orders where the user has access on the organization"""
+        queryset = super().get_queryset()
+        organization_access_filter = {"organization__accesses__user": self.request.user}
+        return queryset.filter(**organization_access_filter)
```

Comment:
```suggestion
        return queryset.filter(organization__accesses__user=self.request.user)
```

This does not work ?

Reply:
It does work indeed :+1: thanks